cime_config/allactive/config_pesall.xml

```diff
@@ -2517,21 +2517,20 @@
       </pes>
     </mach>
   </grid>
-  <grid name="a%ne30np4.pg2_l%r05_oi%IcoswISC30E3r5_r%r05_g%mpas.gis20km_w%null_z%null_m%IcoswISC30E3r5">
-    <mach name="chrysalis">
-      <pes compset="1850_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_MALI_SWAV" pesize="any">
-        <comment>GIS 20km (low-res) testing config</comment>
+  <grid name="a%ne30np4.pg2_l%.+_g%mpas.gis4to40">
+    <mach name="pm-cpu|muller-cpu|alvarez-cpu">
+      <pes compset=".*EAM.+ELM.+MPASSI.+MPASO.+MOSART.+MALI.+SWAV.+" pesize="M">
+        <comment>pm-cpu: GIS 4-to-40km (med-res) baseline config</comment>
         <MAX_MPITASKS_PER_NODE>128</MAX_MPITASKS_PER_NODE>
-        <MAX_TASKS_PER_NODE>128</MAX_TASKS_PER_NODE>
```

Comment:
was this meant to be removed? I'm seeing it default to "MAX_TASKS_PER_NODE" value="256"

Reply:
Sorry for the late response here @darincomeau. Thanks for catching this. No, this was not meant to be removed. I've added it back in now (also to a similar missing line near 2450).